test/src/kava-model.spec.js

```diff
@@ -73,6 +74,7 @@ describe('KavaModel', () => {
     model.getClientVer = () => cv;
     model.getClientTag = () => ct;
     model.getKS = () => ks;
+    model.getVirtualEventId = () => veid;
```

Comment:
```suggestion
    model.getVirtualEventId = () => void;
```

Reply:
'veid' stands for vertualEventId :) see https://github.com/kaltura/playkit-js-kava/pull/109/files/1b9714149ad62c938476ada5312ed9fef29f41e9#diff-e7b75ddd5b46b84b8b502f0da920ad0a2ecde0be40ad52a6f6af1356deb23978R13
it's ok